docs/python/editing.md

```diff
@@ -62,14 +62,15 @@ You can also customize the general behavior of autocomplete and IntelliSense, ev
 
 ### Troubleshooting IntelliSense
 
-For autocomplete and IntelliSense issues, check the following causes:
-
-| Cause | Solution |
-| --- | --- |
-| Pylance seems slow or is consuming too much memory when working on a large workspace. | If there are subfolders you know can be excluded from Pylance's analysis, you can add their paths to the `python.analysis.exclude` setting to see if performance improves. Alternatively, you can try setting `python.analysis.indexing` to `false` to disable Pylance's indexer (**Note**: this will also impact the experience of completions and auto imports. Learn more about indexing in [code analysis settings](/docs/python/settings-reference.md#code-analysis-settings)).  |
-| Pylance is only offering top-level symbol options when adding imports. | Try increasing the depth to which Pylance can index your installed libraries through the `python.analysis.packageIndexDepths`. Check [code analysis settings](/docs/python/settings-reference.md#code-analysis-settings).     |
-| The path to the python interpreter is incorrect | Make sure you selected a valid interpreter path by running the **Python: Select Interpreter** command (see [Environments](/docs/python/environments.md)). |
-| The custom module is located in a non-standard location (not installed using pip). | Add the location to the `python.autoComplete.extraPaths` setting and restart VS Code. |
+For common Intellisense and Python editing issues, check the following causes:
+
+| Problem | Cause | Solution |
+| --- | --- | ---|
+| Pylance seems slow or is consuming too much memory when working on a large workspace. | Pylance analysis is done on all files present in a given workspace.  | If there are subfolders you know can be excluded from Pylance's analysis, you can add their paths to the `python.analysis.exclude` setting. Alternatively, you can try setting `python.analysis.indexing` to `false` to disable Pylance's indexer (**Note**: this will also impact the experience of completions and auto imports. Learn more about indexing in [code analysis settings](/docs/python/settings-reference.md#code-analysis-settings)).  |
```

Comment:
I think the best order for this table would be:
1. The path to the Python interpreter is incorrect
2. Import could not be found *(reportMissingImports)*
3. Pylance is only offering top-level symbol options when adding imports.
4. Pylance seems slow or is consuming too much memory when working on a large workspace.
5. You are unable to install a custom module into your Python project.